packages/package_info/example/pubspec.yaml

```diff
@@ -10,6 +10,7 @@ dependencies:
 dev_dependencies:
   flutter_driver:
     sdk: flutter
+  instrumentation_adapter: "^0.1.3"
```

Comment:
We should figure out if this change needs a bump of the minimally required Flutter SDK

Reply:
Updated it to `flutter: ">=1.6.7 <2.0.0"`